rescript

```diff
@@ -512,59 +512,62 @@ Please pick a different one using the \`-ws [host:]port\` flag from bsb.`);
     buildFinishedCallback(0);
   });
 } else {
-  switch (maybeSubcommand) {
-    case "info":
-    case "clean": {
-      delegateCommand(process_argv.slice(2));
-      break;
-    }
-    case undefined:
-    case "build": {
-      logStartCompiling();
-      delegateCommand(process_argv.slice(2), logFinishCompiling);
-      break;
+  const isDefinitelyBuild =
+    maybeSubcommand === undefined ||
+    (maybeSubcommand === "build" && process_argv.length === 3);
+
+  if (isDefinitelyBuild) {
+    logStartCompiling();
+    delegateCommand(process_argv.slice(2), logFinishCompiling);
+  } else {
```

Comment:
A little bit hacky, but I think better, than nothing. Ideally, to move all the args parsing logic on the js side in one place.

Reply:
https://nodejs.org/api/util.html#utilparseargsconfig will be useful if we accept https://github.com/rescript-lang/rescript-compiler/pull/6429

Reply:
I was actually thinking of using `minimist`, but https://github.com/pkgjs/parseargs also looks good.

Reply:
If we are more tolerant of external dependencies we can even import more, like clipanion :joy: